WooCommerce/WooCommerceUITests/Flows/MockDataReader.swift

```diff
@@ -74,7 +74,7 @@ class GetMocks {
         return updatedData
     }
 
-    static func readNewOrderData() throws -> OrderData {
+    static func readSingleOrderData() throws -> OrderData {
```

Comment:
renamed this from new to single order because in the case of universal link it isn't a new order